src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,98 @@
+package lotto.domain;
+
+import java.util.*;
+
+public class Lotto {
+    public static final String IS_NOT_LOTTO_SIZE = "번호가 6개가 아닙니다.";
+    private static final String INPUT_IS_NOT_NUMBER = "입력된 값이 숫자가 아닙니다.";
+    public static final String IS_NOT_LOTTO_NUMBER = "로또번호가 1~45가 아닙니다.";
```

Comment:
```suggestion
    private static final String IS_NOT_LOTTO_SIZE = "번호가 6개가 아닙니다.";
    private static final String INPUT_IS_NOT_NUMBER = "입력된 값이 숫자가 아닙니다.";
    private static final String IS_NOT_LOTTO_NUMBER = "로또번호가 1~45가 아닙니다.";
```
리뷰 코멘트에서 한군데만 적어둔 코멘트들이 있는데 중복 코멘트를 남기지 않기 위해서 해당 부분에만 코멘트를 남기는 것이므로 가능하면 리뷰에 대해서 전체적으로 변경할 점이 있는지 점검부탁드려요 :)